src/components/Newfeature.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const NewFeature = () => {
+    return (
+        <div className="">
+            <h1 className="">This is a new feature!</h1>
```

Comment:
please write here "This is a super cool feature new feature!"

Reply:
ok